app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :users
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_post_counter
+    users.posts_counter = post.count
+  end
```

Comment:
- [ ] Great job creating a method that updates the posts counter for a user. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code should look like this 👇🏻 :


   ```ruby
     # app/models/post.rb
     after_save :update_post_counter
   
     private
   
     def update_post_counter
       author.increment!(:posts_counter)
     end
   ```